### PR TITLE
Whores and Drugs Update

### PR DESCRIPTION
**Pull request by OctusGit**
## About The Pull Request

Remaps the bathhouse for Dunn Manor.
Gives the drug market and instruments tab to the bath house, giving them PURITY 2.0, similar to the merchants shop but only for drugs and instruments. And a new machine called NARCOS, that liquidates any drugs to cash.
Cleans up and reorganizes the merchant shop (They have Jewelry now as a category)
Sprites for the NARCOS and PURITY machines are WIP...

## Why It's Good For The Game

Bathhouse lacks any real substance aside being glorified ERP hub which is pretty boring in the overall grand span of the game. Letting the bathhouse have control over narcotics and even instruments gives them more of a position to be a useful group toward the town for addicts and rich nobles wanting to get fucked up. It also allows the Niteman to be more of a noble, and overall increases their ability to negotiate and offer other services aside baths and sex.

Also old map was kind of crap, too big with alot of nothing.. This one is more compact with actual substance in each room.
laAWcedNks

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by bowlofcereal** [comment]
You're so fucking real for this.

File: code/modules/cargo/packsrogue/luxury.dm [hunk 1]
```diff
@@ -103,7 +28,23 @@
...
+
+/datum/supply_pack/rogue/luxury/gold_plaque_belt
+	name = "Gold Plaque Belt"
+	cost = 60
+	contains = /obj/item/storage/belt/rogue/leather/plaquegold
+
+/datum/supply_pack/rogue/luxury/spectacles_golden
+	name = "Golden Spectacles"
+	cost = 40
+	contains = /obj/item/clothing/mask/rogue/spectacles/golden
```
Comment (BadAtThisGame302) [inline code/modules/cargo/packsrogue/luxury.dm:44]: Silver/Gold plaque belts and the golden spectacles all exist on apparel.dm

**Comment by RoboticVodka** [comment]
Downgrade on the mapping side. Notably the back way into the inn's kitchen was removed if you wanted to run food through there in partnership with the inn.
bbbc2dd243c36a16caa9d1b84ebbebd2
d3ff54c3a64cb39dfe92ac9237fd33f9

These rooms serve no purpose or at least the same purpose as the old changing room(No one used) but take up more space. If they're meant to be private bathing areas they should probably have buckets to actually pour over yourself maybe stools but still that's taking away from nitemaidens wanting to pull them away to the more private rooms and tempt them further there.
bdb64d3283d86b41ceba5fc376338dcf

There's also no options for more personal seating aside from the single booth now. Where before couches and tables could be used that were littered around. Even the group "party" room had more than a single central table

This area is just wasted space.
467c632c6a6116914989969de42c7c30

The nitemaiden bunkroom removes the four randomly colored dresses of different styles that the maidens could have used to have slightly different styles to replace them with the same dress they already have but in red and not enough for all four. The nitemaidens shared bedrooms as well served as rooms they could take customers back to I believe given how they're connected to the actual private baths so they wouldn't have to fuck on a couch if for some reason they chose to.
2887fcae7dbf3553381c3007027a346f
8cf10dc24a7dd896effba4f61638277c


Floor could use more variation multiple different carpet types were used in the old one and other tile types to help it look less flat. Map as is makes it smaller but still wastes considerable space while taking away amenities from the old map and visually just seems like a downgrade.

File: code/modules/roguetown/roguemachine/merchant.dm [hunk 2]
```diff
@@ -287,7 +287,7 @@
 
 	contents += "</center><BR>"
 
-	var/list/unlocked_cats = list("Apparel","Armor","Consumable","Luxury","Tools","Seeds","Weapons")
+	var/list/unlocked_cats = list("Apparel","Armor","Consumable","Jewelry","Tools","Seeds","Weapons")
```
Comment (BadAtThisGame302) [inline code/modules/roguetown/roguemachine/merchant.dm:290]: You need to change the file name (everything that's luxury to Jewelry) on luxury.dm for it to work.

**Comment by OctusGit** [comment]
> Downgrade on the mapping side. Notably the back way into the inn's kitchen was removed if you wanted to run food through there in partnership with the inn. 

This has never been a thing, so its removal has no merit personally to me, or anyone who thinks this will ever happen.
> 
> These rooms serve no purpose or at least the same purpose as the old changing room(No one used) but take up more space. If they're meant to be private bathing areas they should probably have a water bin and buckets to actually pour over yourself maybe stools but still that's taking away from nitemaidens wanting to pull them away to the more private rooms and tempt them further there. 

Fair criticism, ill change them to something else and see what I can come up with.
> 
> There's also no options for more personal seating aside from the single booth now. Where before couches and tables could be used that were littered around. Even the group "party" room had more than a single central table

Another fair critique sure, I doubt there ever needs to be more then one table but I can understand having optons.
> 
> This area is just wasted space. 

Yep, fair critique.
> 
> The nitemaiden bunkroom removes the four randomly colored dresses of different styles that the maidens could have used to have slightly different styles to replace them with the same dress they already have but in red and not enough for all four. The nitemaidens shared bedrooms as well served as rooms they could take customers back to I believe given how they're connected to the actual private baths so they wouldn't have to fuck on a couch if for some reason they chose to. 

Maidens always fuck in the pool/couch area. Ive never seen nor believe a maiden will fuck someone on the bed when the standard has always ben set. Dress's sure I can fixit, but bed critique has no merit and I disagree.
> 
> Floor could use more variation multiple different carpet types were used in the old one and other tile types to help it look less flat. Map as is makes it smaller but still wastes considerable space while taking away amenities from the old map and visually just seems like a downgrade.

I can add different floors then.

**Comment by OctusGit** [comment]
Map updated, with changes from the critique, please view in the new map image.

File: code/modules/cargo/packsrogue/luxury.dm (hunk 1 above)
Comment (OctusGit) [reply code/modules/cargo/packsrogue/luxury.dm]: fixed

File: code/modules/roguetown/roguemachine/merchant.dm (hunk 2 above)
Comment (OctusGit) [reply code/modules/roguetown/roguemachine/merchant.dm:290]: fixed

**Comment by NPC1314** [comment]
Ok so we have moved to TGM now, thats the "modern" TG system where the map is basically modular so people should be able to merge maps without issues as long theyre not makking the exacxt same spots.

What this means is this needs converting to TGM too , using StrongDMM for example.

**Review by Optimism333 — COMMENTED** [review]
Whenever you can, please do fix the merge conflict and make sure your map changes are saved in the TGM format. Otherwise this looks ready to go.

**Comment by RoboticVodka** [comment]
efbd4bcf0a4a543fe8bc82c3902f8470-1
These rooms being copies of each other but one with carpet seems a bit lazy I'd suggest something like below give it some more flavor as looking upscale and the 1 on 1 seating could be like a sort of 'noble' meeting area or a soothesayer's ideal little 1 on 1 room if someone wants to roleplay it or we ever add some kind of tarot or divination mechanics.
6067d7b776e7b98ceea0667a50b8e251-1
/
c47e88fa3de5ef4a0b707868af92f17c
This rooms looks excessively sterile especially with how the tables are lined up. It also lacks an SCOM and HERMES like the original room. Maybe the HERMES isn't needed but the SCOM should absolutely be included if this is intended to be the main meeting area so someone can shout and not have to barge down all the time. Suggest something like this with similar seating capacity but not as uniform.
ed91f2157462c396c8ac91ae66f28970

Also just a minor nitpick maybe give the beds bedsheets.

**Comment by OctusGit** [comment]
Ive readded the pathway between the baths and the inn underground with two alt routes to choose. Useful for navigation and also making the area less of a deadend. Also updated format to TGM.

**Review by Optimism333 — APPROVED** [review]
Conversion to TGM... successful...

Remapping documented...

Conflicts... resolved...

We are going live...

**Pull request merged**